TF_Settings_and_Tooling_Unity/Assets/TouchFree/SettingsUI/Scripts/Configuration/Screens/CameraPreviewScreen.cs

```diff
@@ -26,6 +27,13 @@ public class CameraPreviewScreen : MonoBehaviour
 
     void OnEnable()
     {
+        Camera.main.gameObject.GetComponent<Leap.Unity.LeapImageRetriever>().enabled = false;
+        Camera.main.gameObject.GetComponent<Leap.Unity.LeapImageRetriever>().enabled = true;
```

Comment:
this could use the newly added leapImageRetriever reference on line 11 rather than using GetComponent

Reply:
I've pushed up a change to fix this